setup.cfg

```diff
@@ -1,6 +1,56 @@
-[bdist_wheel]
-universal=1
+[metadata]
+name = odltools
+summary = Tools for troubleshooting OpenDaylight
+description-file =
+    README.rst
+author = Sam Hague
+author-email = shague@gmail.com
+home-page = http://github.com/shague/odltools
+license =  Apache-2
+classifier =
+        Development Status :: 1 - Planning
+        Environment :: Console
+        Environment :: OpenStack
+        Intended Audience :: Developers
+        Intended Audience :: Information Technology
+        Intended Audience :: System Administrators
+        License :: OSI Approved :: Apache Software License
+        Natural Language :: English
+        Operating System :: OS Independent
+        Programming Language :: Python
+        Programming Language :: Python :: 2
+        Programming Language :: Python :: 2.7
+        Programming Language :: Python :: 3
+        Programming Language :: Python :: 3.3
+        Programming Language :: Python :: 3.4
+        Programming Language :: Python :: 3.5
+        Programming Language :: Python :: 3.6
+        Topic :: Software Development
+        Topic :: System :: Networking
+        Topic :: System :: Systems Administration
+        Topic :: Utilities
 
-[egg_info]
-#tag_build = dev
-tag_build =
+[global]
+setup-hooks =
+    pbr.hooks.setup_hook
+
+[entry_points]
+console_scripts =
+    odltools = odltools.__main__:main
+
+[files]
+packages =
+    odltools
+
+[wheel]
+universal = 1
+
+[pbr]
+skip_authors = True
+skip_changelog = True
+
+[build_sphinx]
+project = 'ODL Tools'
+version = 1.0
```

Comment:
still at 0.1.x version right now

Is there a way to pull the __version__ from the __init__.py file as it is with setuptools?

Reply:
yeah will we can remove this and conf.py pull in the version